src/patternfly/components/TreeView/examples/TreeView.md

```diff
@@ -112,54 +112,54 @@ beta: true
       {{#> tree-view-list newcontext}}
         {{#> tree-view-list-item tree-view-list-item--IsExpandable="true" tree-view-list-item--IsExpanded="true"}}
-          {{> tree-view-node tree-view-node--type="div" tree-view-node--HasCheckbox="true" tree-view-node--text="Application 1"}}
+          {{> tree-view-node tree-view-node--HasToggle="true" tree-view-node--id="node-2" tree-view-node--type="div" tree-view-node--HasCheckbox="true" tree-view-node--IsChecked="true" tree-view-node--text="Application 1"}}
           {{#> tree-view-list newcontext}}
             {{#> tree-view-list-item}}
-              {{> tree-view-node tree-view-node--type="div" tree-view-node--HasCheckbox="true" tree-view-node--text="Settings"}}
+              {{> tree-view-node tree-view-node--id="node-3" tree-view-node--type="div" tree-view-node--HasCheckbox="true" tree-view-node--IsChecked="true" tree-view-node--text="Settings"}}
             {{/tree-view-list-item}}
             {{#> tree-view-list-item}}
-              {{> tree-view-node tree-view-node--type="div" tree-view-node--HasCheckbox="true" tree-view-node--HasCheckboxSelected="true" tree-view-node--text="Loader"}}
+              {{> tree-view-node tree-view-node--id="node-4" tree-view-node--type="div" tree-view-node--HasCheckbox="true" tree-view-node--IsChecked="true" tree-view-node--IsChecked="true" tree-view-node--text="Loader"}}
             {{/tree-view-list-item}}
           {{/tree-view-list}}
         {{/tree-view-list-item}}
         {{#> tree-view-list-item tree-view-list-item--IsExpandable="true" tree-view-list-item--IsExpanded="true"}}
-          {{> tree-view-node tree-view-node--type="div" tree-view-node--HasCheckbox="true" tree-view-node--text="Application 2"}}
+          {{> tree-view-node tree-view-node--HasToggle="true" tree-view-node--id="node-5" tree-view-node--type="div" tree-view-node--HasCheckbox="true" tree-view-node--text="Application 2"}}
           {{#> tree-view-list newcontext}}
             {{#> tree-view-list-item tree-view-list-item--IsExpandable="true"}}
-              {{> tree-view-node tree-view-node--type="div" tree-view-node--HasCheckbox="true" tree-view-node--text="Settings"}}
+              {{> tree-view-node tree-view-node--HasToggle="true" tree-view-node--id="node-6" tree-view-node--type="div" tree-view-node--HasCheckbox="true" tree-view-node--text="Settings"}}
             {{/tree-view-list-item}}
             {{#> tree-view-list-item tree-view-list-item--IsExpandable="true" tree-view-list-item--IsExpanded="true" tree-view-node--text="Current"}}
-              {{> tree-view-node tree-view-node--type="div" tree-view-node--HasCheckbox="true"}}
+              {{> tree-view-node tree-view-node--HasToggle="true" tree-view-node--id="node-7" tree-view-node--type="div" tree-view-node--HasCheckbox="true" tree-view-node--IsChecked="true"}}
```

Comment:
Are these checked checkboxes correct? Wouldn't loading app 1 and loader app 3 also be checked if the parent was? I think in this case the way it is, current would be indeterminate? Also should they all be either "loading" or "loader", instead of both, since it's a sequential list?

<img width="346" alt="Screen Shot 2020-08-28 at 4 47 10 PM" src="https://user-images.githubusercontent.com/35148959/91617721-38ed0600-e94e-11ea-8912-84f2312e5aa9.png">